scaling_performance/optimizing_storage.adoc

```diff
@@ -0,0 +1,114 @@
+[[scaling-performance-optimizing-storage]]
+= Optimizing Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+
+== Optimizing Storage
+
+Docker stores images and containers in a graph driver (a pluggable storage
```

Comment:
Please replace all instances of OverlayFS with Overlay.  The upstream kernel module was renamed.